### ISSUE TITLE
PFE, Performance, Tiller deployments remains after deleting CheCluster

### ISSUE THREAD

**Issue by makandre** [eclipse/codewind#1468]
**Codewind version:**
**OS:**

**Che version:**
**IDE extension version:**
**IDE version:**
**Kubernetes cluster:** OCP 4.2

**Description:**

Started a Codewind workspace in Che, then I deleted the CheCluster.  The PFE, Performance and Tiller deployments are still left running.

**Steps to reproduce:**

1. Start a Codewind workspace
2. Without stopping the workspace, go to Installed Operators section in OCP UI
3. Locate the Che operator, drill down to the CheCluster
4. Delete the CheCluster

**Workaround:**

**Comment by johnmcollier** [comment]
@makandre Were there Che workspace resources (pods or PVCs) leftover too? If so, unfortunately there's not much we can do here. Deleting a `CheCluster` resource does not ensure that running workspaces are stopped or cleaned up.

It's recommended that users stop the Che workspace(s) first, before deleting the CheCluster custom resource. The tiller sticking around will be fixed by https://github.com/eclipse/codewind/pull/1358

Perhaps this can be a requirement we take to the Che folks?

**Comment by makandre** [comment]
Yes, Che workspace resources do get left behind too; I was just hoping we could do something to stop PFE and Perf but if that's a limitation with the way Che behaves then that's fine

**Comment by johnmcollier** [comment]
Yeah, from a Codewind side, not sure what we can do, since we're tied to the workspace resources. But I'll at least see if Che has an issue or feature open to address this, and if not, open one on their side.
